crates/pathfinder/src/storage/state.rs

```diff
@@ -1095,10 +1015,15 @@ impl StarknetEventsTable {
                 .collect();
 
             let keys = row.get_ref_unwrap("keys").as_str().unwrap();
+
+            let mut temp = [0u8; 32];
+
             let keys: Vec<_> = keys
                 .split(' ')
                 .map(|key| {
-                    let key = StarkHash::from_be_slice(&base64::decode(key).unwrap()).unwrap();
+                    base64::decode_config_slice(key, base64::STANDARD, &mut temp).unwrap();
```

Comment:
This should use the amount of bytes... Should be a `#[must_use]` over at base64 actually. Theoretically we could have shorter strings there but not necessarily.

Reply:
I don't think that a `#[must_use]` over at decode_config_slice would had helped since `.unwrap` uses the value just as well.